vignettes/bds_finding.Rmd

```diff
@@ -288,25 +341,30 @@ adeg <- derive_param_qtcf(
  
 ```
 
-```{r eval=TRUE, echo=FALSE}
-advs <- filter(advs, !(PARAMCD %in% c("MAP", "BSA")))
+All parameters now added so can add other parameter information from lookup
+table containing ``PARAMCD`` information.
```

Comment:
Here also i would rephrase: When all `PARAMCD` have been derived and added to the dataset, the other information from the look-up table (PARAM, PARAMCAT1,...) could be added.

Reply:
Done